build/configs/qemu/README.md

```diff
@@ -35,6 +35,14 @@ qemu-system-arm -M lm3s6965evb -kernel ../build/output/bin/tinyara -nographic
 make download
 ```
 
+## How to debug
```

Comment:
Please add description for this.
Without any description, "How to debug" is not appropriate.
i.e.
```
To debug Tizen RT on QEMU, GDB should be connected through below commands.
```